presto-spark-base/src/main/java/com/facebook/presto/spark/PrestoSparkConfig.java

```diff
@@ -37,6 +39,7 @@
     private DataSize sparkBroadcastJoinMaxMemoryOverride;
     private boolean smileSerializationEnabled = true;
     private int splitAssignmentBatchSize = 1_000_000;
+    private double memoryRevokingThreshold;
```

Comment:
Have no idea what a proper default value should be,  I am planning to use session property to optimize this for different operators.